src/docs/development/tools/devtools/memory.md

```diff
@@ -5,41 +5,68 @@ description: Learn how to use the DevTools memory view.
 
 ## What is it?
 
-Allocated Dart objects created using a class constructor
-(for example, by using `new MyClass()` or `MyClass()`)
-live in a portion of memory called the heap.
+Allocated Dart objects created using a class constructor (for
+example, by using `new MyClass()` or `MyClass()`) live in a partion
+of memory called the heap that is managed by the Dart VM (virtual
+machine). 
 
 ## DevTools memory page
 
 DevTools Memory page lets you peek at how an isolate is using
-memory at a given moment. Using Monitor Allocations and
-Reset options, you can analyze accumulator counts. The accumulators
-can be used to study the rate of memory allocations,
-if you suspect your application is leaking memory or has
-other bugs relating to memory allocation.
-
-A Snapshot is a complete view of all objects in the Dart memory heap.
-Each time a snapshot is taken, an analysis is performed over the memory
-data.  The analysis attempts to identify any memory patterns that may
-cause leaks or lead to application crashes. For example, loading large
-assets for thumbnail-sized images in inefficient, memory usage can
-be improved by loading smaller assets or adjusting the cacheWidth/cacheHeight
-to decode an image to a smaller size reducing the memory usage of the
-ImageCache. The analysis catches issues like this.
-
-Memory profiling consists of six parts, each increasing
-in granularity:
+memory at a given moment. 
+
+Memory profiling consists of six parts, each increasing in granularity:
 
 * Memory overview chart
 * Android-only memory chart (via Android's ADB tool)
-* Event timeline (for example, garbage collection (GC)
+* Event timeline e.g., garbage collection (GC)
   events, user memory tooling interactions)
-* Monitoring of Dart total instances,
+* Monitor of Allocations&mdash;total instances,
   size and accumulator monitoring to detect memory leaks
+* Track when memory for a particular class is contructed
+  (new instance)&mdash;stack trace of your code
 * Analysis of a Snapshot to detect possible memory problems
   or Flutter gotchas
-* Snapshots of all live memory class instances and sentinels
-  (instances to be GC'd)
+* Snapshot of all live memory class instances and sentinels
+  (instances to be GC'd) and ability to inspect the values
+  of the objects
+
+At the top-level, when the memory tab is selected memory statistics from
+the VM are collected. These statistics are displayed in the two overview
+charts (Dart memory and Android-only) the collection of general memory
+usage e.g., total heap used, external heap, maximum heap capacity,
+Resident Set Size (RSS). As you interact with your application various
+events are detected e.g., memory GC (garabage collection),Flutter events,
+user fired events (using the ```dart:developer``` package) are collected
+in the same timeline as the memory statistics. All of this collected
+statistics and events are displayed in charts (see Anatomoy of Memory).
```

Comment:
I think this whole section can be replaced with:

When the memory tab is opened for the first time in DevTools, memory statistics from the VM are collected. The memory usage data collected consists of total heap size used, external heap size used, maximum heap capacity, and resident set size (RSS). The usage statistics are polled and displayed in the memory overview chart.

As you interact with your application, various events are also detected and displayed in the overview chart (e.g. GC's, Flutter events, user-fired events from dart:developer).